src/utils/Card.js

```diff
@@ -77,8 +77,145 @@ export function cardsAreEquivalent(a, b) {
   );
 }
 
-export function getCmc(card) {
-  return card.cmc !== undefined ? card.cmc : card.details.cmc;
-}
+export const cardTags = (card) => card.tags;
+
+export const cardFinish = (card) => card.finish;
+
+export const cardStatus = (card) => card.status;
+
+export const cardColorIdentity = (card) => card.colors ?? card.details.color_identity;
+
+export const cardCmc = (card) => card.cmc ?? card.details.cmc;
+
+export const cardId = (card) => card.cardID ?? card.details._id;
+
+export const cardType = (card) => card.type_line ?? card.details.type;
+
+export const cardRarity = (card) => card.rarity ?? card.details.rarity;
+
+export const cardAddedTime = (card) => card.addedTmsp;
+
+export const cardImg = (card) => card.imgUrl ?? card.details.image_normal ?? card.details.image_small;
+
+export const cardNotes = (card) => card.notes;
+
+export const cardColorCategory = (card) => card.colorCategory ?? card.details.color_category;
+
+export const cardPrice = (card) =>
+  cardFinish() === 'Foil'
+    ? card.details.price_foil ?? card.details.price
+    : card.details.price ?? card.details.price_foil;
+
+export const cardNormalPrice = (card) => card.details.price;
+
+export const cardFoilPrice = (card) => card.details.price_foil;
+
+export const cardIsFullArt = (card) => card.details.full_art;
+
+export const cardCost = (card) => card.details.parsed_cost;
```

Comment:
```suggestion
export const cardParsedCost = (card) => card.details.parsed_cost;
```

Reply:
If we don't have an unparsed cost I don't think adding in parsed adds that much useful information and cardCost is so much shorter. Can still make the change though if you feel strongly about it.